internal/server/statvar/statvar_group.go

```diff
@@ -137,12 +137,12 @@ func GetStatVarGroup(
 
 	var result *pb.StatVarGroups
 	if cache == nil {
-		// Read stat var group cache from the first table, which is the most
-		// preferred cache BigTable. Since stat var schemas are rebuild with every
-		// group, so no merge needed.
+		// Read stat var group cache from the frequent import group table. It has
+		// the latest and trustworthy stat var schemas and no need to merge with
+		// other import groups.
 		btDataList, err := bigtable.Read(
 			ctx,
-			store.BtGroup,
+			bigtable.GetFrequentGroup(store.BtGroup),
```

Comment:
Just to check, are there other cache types where we can read from the best one (whichever it is) without reading everything and picking the first?

Reply:
Seems only the stat var hierarchy is standalone; Other caches all span schema and data/stat..